dataactvalidator/config/sqlrules/d17_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,11 @@
+-- LegalEntityForeignPostalCode is optional for foreign recipients (i.e., when LegalEntityCountryCode != USA)
```

Comment:
If there are no rules we're writing for this to test (we don't need to, it can literally be anything) then this comment isn't needed. We only need the comment that describes the sql.